eve/buildplan/buildplan/dsl/artifacts.py

```diff
@@ -0,0 +1,79 @@
+from collections import Mapping
```

Comment:
I would have done the split into a dedicated commit, because here it's hard to tell the difference between the new code that we should review and the old code that is simply moved around.